.github/workflows/frontend-check.yaml

```diff
@@ -0,0 +1,55 @@
+name: Cypress tests
+on: 
+  push:
+    # branches:
+    #   - main
+    # tags:
+    #   - v*
```

Comment:
That's a valid question. I'm not sure we want to trigger on a `push` to *any* branch? But maybe? Do we need to run tests again on defining a tag? I like testing on a merge to `main` (or, in the short term, `revamp`); we don't usually run with additional branches, so this may not matter much.

Reply:
I copied this from an arcaflow github workflow, but I'm open to changing it. I wasn't sure what we'd want to do here

Reply:
I'm inclined to say we don't need to re-run the CI on tagging; `main` should be kept clean by the PR testing and merge testing... in which case we should remove the commented `tags` phrase entirely.

While I don't think the `branches` phrase here really matters as we don't routinely use long-lived branches, if we're not going to use it we should remove it instead of commenting it. I'd be OK with either removing it or using it with `main` and (temporarily) `revamp`.